src/easylink/pipeline_graph.py

```diff
@@ -47,53 +50,81 @@ def update_slot_filepaths(self, config: Config) -> None:
         ):
             for edge_idx in self[source][sink]:
                 if edge_attrs["output_slot"].name == "all":
-                    self[source][sink][edge_idx]["filepaths"] = [
-                        str(path) for path in config.input_data.to_dict().values()
-                    ]
+                    self[source][sink][edge_idx]["filepaths"] = tuple(
```

Comment:
Are you making this a tuple so that it's immutable?

Reply:
correct